### DISCUSSION TITLE
Items not on the PCBWAY assembly and Gizmo Case?

### DISCUSSION THREAD

**Discussion by MKS2345** [gizmo-platform/gizmo#37] [Q&A] [answered]
Hi, I am looking into the gizmo for my school, We are currently looking into ordering a limited number of the PCB boards assembled from PCBWAY and ran into a few questions. 

On the rendering of the board on the link supplied on the gizmo website, there are 2 items not shown on the board. From what I can tell, they are the Ethernet Jack and the chip marked as T1. I am wondering if these will still be on the board from PCBWAY or whether I will need to source them and solder them on manually. 

On the Gizmo website, it says that we would need to source our own case, I have also seen photos of the Gizmo with a case that looks to be 3d printed, are there files for this case that I could 3d print?

Lastly, this might be a dumb question but is there a way to view the output log of the Gizmo from the FMS or Driver Station, If not is is possible to view it over a wired connection? I have looked around and have not found anything on this topic. 

Any help would be greatly appreciated!

**Comment by the-maldridge** [comment, marked as answer]
Apologies for the slow reply, I thought I'd hit enter but apparently dreamed that.

The two parts not shown on the board are part of the BOM and will be sourced/assembled, but the vendors do not provide 3d models for them so they don't show up in the rendering.  Its an ongoing item we'd like to fix but 3d component modeling isn't a skill we have in-house.

There are a few cases that are 3d printable, you can find them on Printables.  these are the 2 that I'm aware of:

  * https://www.printables.com/model/1026594-gizmo-verson-10-case-2
  * https://www.printables.com/model/997498-best-robotics-gizmo-v1-case

I've personally printed the first one and can say its very solid.  The second uses less material, but I haven't tried it myself.

The log output is not currently streamed to the FMS, but we can take that as a feature request if you'd like to open an issue for it.  You can view the Driver's Station logs by connecting an HDMI display to it, it prints them to the default console.  The Gizmo itself prints an event log to the system processor's USB port at `9600,8,n,1`.  The one limitation we'd have with streaming logs to the FMS is that of course it wouldn't work if the FMS can't be reached (which would somewhat blunt the usability of the logs).  If you believe there's sufficient value given this limitation, feel free to make the case.

**Reply by MKS2345** [reply]
Thank you,

This reply was very helpful, we were wondering if it could be output to the FMS because we are planning on using it in the classroom environment and it would be nice to not have to source displays for each DS, If it is also output through USB though that is sufficient for our use case.
